config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   
   get 'server', to: 'server#index'
+  get 'notifications', to: 'notifications#notifications'
```

Comment:
Okay cool, I really wanted to do that actually but I didn't wasn't sure if that would be best practice. It really didn't make sense to me that a controller that was used to request both notifications and actions was called `notifications`
I'll do that as soon as I get some time to code